ISSUE_TEMPLATE/next.md

```diff
@@ -58,10 +60,10 @@ From a checkout of this repo:
 fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/prod/streams/next/releases.json  -output-file=streams/next.json -pretty-print
 ```
 
-- [ ] Add a rollout.  For a 48-hour rollout starting at 10 AM ET, run:
+- [ ] Add a rollout.  For a 48-hour rollout starting at 10 AM ET the same day, run:
```

Comment:
when I first read this I read it as "you must do 10AM the same day". Let's make it a little more clear:

```suggestion
- [ ] Add a rollout.  For example, for a 48-hour rollout starting at 10 AM ET the same day, run:
```